TESTS/network/wifi/wifi_scan.cpp

```diff
@@ -54,12 +54,16 @@ void wifi_scan(void)
         if (strcmp(MBED_CONF_APP_WIFI_SECURE_SSID, ssid) == 0) {
             secure_found = true;
             TEST_ASSERT_EQUAL_INT(NSAPI_SECURITY_WPA2, security);
```

Comment:
Should this be changed also to WPA/WPA2?

Reply:
Yes, it should and I'll change it.

Reply:
Security to check against should be defined in the .json which also defines the wlan ap to connect to.